blockage/plugins.py

```diff
@@ -2,6 +2,22 @@
     import httplib
 except ImportError:
     import http.client as httplib
+
+try:
+    unicode = unicode
```

Comment:
sure. It's an old PR, I don't remember where this comes from. Google says this code is pretty common, it's the same in other projects. Probably I was porting other projects at the same time and this code was there...